helpers/custom_ners.py

```diff
@@ -55,3 +52,12 @@ def get_redis_filters(query):
         return " | ".join(filters)
     else:
         return None
+
+
+if __name__ == "__main__":
+
+    nlp = English()
+    ruler = nlp.add_pipe("entity_ruler", config={"validate": True})
```

Comment:
What was this for?

Reply:
Essentially, it loads the SEC data and converts it into a set of NER components that we hydrate, and can then be used for extracting filters from the question.